examples/client/lwm2mclient.c

```diff
@@ -95,6 +95,7 @@ lwm2m_object_t * objArray[OBJ_COUNT];
 # define BACKUP_OBJECT_COUNT 2
```

Comment:
same goes for this and all other variable definitions. What is their role in the codebase? What is their other possible  default values?

Reply:
Will be done in next commit